modulemd/v2/modulemd-profile.c

```diff
@@ -62,8 +62,8 @@ modulemd_profile_copy (ModulemdProfile *self)
 
   p = modulemd_profile_new (modulemd_profile_get_name (self));
 
-  modulemd_profile_set_description (p,
-                                    modulemd_profile_get_description (self));
```

Comment:
Using `NULL` in the tests would mean that their success/failure would be depending on the actual locale in tests where we build the object with a translation.
Maybe it's an idea to instead just pass an explicit `"C"`, just to make sure that any tests that test with translated entries, the primary test isn't ending up failing on the `robots_TABLES` locale?

Reply:
Or, alternatively, we could make sure to set a locale for the tests in the meson configuration?

Reply:
Yeah, having meson set `LC_ALL=C` in the environment seems like the easiest approach.